appinventor/blocklyeditor/src/typeblock.js

```diff
@@ -141,7 +141,6 @@ Blockly.TypeBlock.prototype.handleKey = function(e){
     // Don't steal input from Blockly fields.
     if (e.target != this.ac_.getTarget() &&
       (e.target.tagName == 'INPUT' || e.target.tagName == 'TEXTAREA')) return;
-    if (e.altKey || e.ctrlKey || e.metaKey || e.keyCode === 9) return; // 9 is tab
```

Comment:
Here you drop the semantics of handling the alt/ctrl/meta keys.

Reply:
i moved these to the switch below

Reply:
One concern that I have that I will need to test is whether the browser reports the boolean values differently from the key codes. For example, I have a script on one computer that swaps some of these keys and my concern is that keyCode matches the physical key but the flag reflects the logical key. If everything checks out, then it would make sense to keep them in the switch.

Reply:
So should i make those back?

Reply:
Hold off on making the change. Let me test it first and if it behaves the same way as the previous version w.r.t. meta, ctrl, and alt then I will merge it as is.